test/blackbox/AsyncAPI.spec.ts

```diff
@@ -1,7 +1,8 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import { JsonSchemaInputProcessor } from '../../src/processors/JsonSchemaInputProcessor';
-describe('AsyncAPI JSON Schema file', function() {
+
+describe.skip('AsyncAPI JSON Schema file', function() {
```

Comment:
This needs to be unskipped

Reply:
AsyncAPI should be skipped, because we don't test it 😄

Reply:
I mean, running the file through the processor is kinda a test in it self, but fair 😄

Reply:
Ahh you added expect, nwm 😄